### DISCUSSION TITLE
华尔街日报的一篇人物报道：凤凰城NBA球队太阳队的老板想处处名利双收

### DISCUSSION THREAD

**Discussion by yibingbingwu** [RoadSideUniversity/ClubHouse2024#31] [General]
刚刚读完[这篇文章](https://github.com/RoadSideUniversity/ClubHouse2023Nov/files/13368315/wsj_article.pdf)
- 中性报道
- 特色的美国企业家
- 典型的年轻有为的犹太人
- 房贷市场供方一瞥

中性报道：
故事简介不加渲染：一个靠批发房贷的[年轻亿万富翁](https://en.wikipedia.org/wiki/Mat_Ishbia)，最近刚买了凤凰城NBA球队太阳队。他号称要今年夺冠，而且年年夺冠。

特色的美国企业家
他的公司在一代人家族从无到有，到今天上市后，父兄三人都是亿万富翁。
我行我素，出台好几个极有争议的措施：竞争对手深恨之，雇员无尽抱怨之
看重回头客：“Ishbia and other executives say big profits will come when the mortgage market roars back and loyal customers return to do more business.”

典型的年轻有为的犹太人
这个人才43岁，从一个挺一般的的公立大学毕业，靠刻苦和敢冒险，20年之内，不靠什么高科技，就功名成就。
从小喜欢打篮球，本事不够上场，但只要有比赛就去，不管哪里。毕业后甘作底薪球队教练。
敢冒风险，在当今房贷利率很高的市场下，仍然敢于砍价扩招。
一旦房市逆转，肯定又是大发一笔。否则接着当教练去。“玩儿的就是心跳”
经营篮球队同样风格

房贷市场供方一瞥
好几个房贷批发商都上市了
“UWM’s push to grab market share last year by offering better rates than its competitors took a bite out of its margins, which dropped to as low as 0.51% on each loan. Margins rebounded to 0.97% in this year’s third quarter. ”
又是个小利大批量生意

**Comment by Lewisrxliu** [comment]
时代成就创业，冒险成就梦想
